### PR TITLE
Added informations to home page.

### PR DESCRIPTION
**Pull request by kdmnk**

### CODE REVIEW

File: resources/views/home.blade.php [hunk 1]
```diff
@@ -16,6 +16,38 @@
                 <p>@lang('general.choose_from_menu')</p>
             </div>
         </div>
+        <div class="card">
```
Comment (luksan47) [inline resources/views/home.blade.php:19]: let's hide this div if there are no news

File: resources/views/home.blade.php [hunk 2]
```diff
@@ -16,6 +16,38 @@
...
+            <div class="card-content">
+                <span class="card-title">@lang('general.information')</span>
+                <form id="info_form" method="POST" action="{{ route('home.edit') }}">
+                    @csrf
+                    <p id="info_text">{{ $information ?? ""}}</p>
+                    <div id="info_input"></div>
+                </form>
+            </div>
+        </div>
+        @if(Auth::user()->hasElevatedPermissions() || Auth::user()->hasRoleBase(\App\Models\Role::STUDENT_COUNCIL))
```
Comment (luksan47) [inline resources/views/home.blade.php:29]: Add it only to student council.

**Review by luksan47 — APPROVED** [review]
lgtm, but seeding did not work for me.

**Comment by kdmnk** [comment]
What do you think now, @luksan47 ?

**Pull request merged**